.github/workflows/desktop.yml

```diff
@@ -103,7 +103,7 @@ jobs:
           LANG: en_US
         run: |
           cd build
-          ctest --verbose
+          ctest --verbose -E "firestore_(objc_spec|objc_integration)_test"
```

Comment:
I think it would be better to exclude them via https://github.com/firebase/firebase-cpp-sdk/blob/master/cmake/CTestCustom.cmake.

Either way, putting a comment explaining why it is being excluded would be good.

Reply:
Yeah, I was contemplating where to put this. Ended up going with this approach because it is more visible and will be a reminder on every workflow update/run. But I noticed that there are few tests already explicitly disabled in CTestCustom.cmake. So maybe having one list of all disabled tests is better. What do you think?

Reply:
Done!